File: spring-boot-react-master/spring-boot-react-master/spring-boot-react/src/main/jsx/Employee.js
import React from 'react';

const Employee = ({ employee }) => {

    return (
        <tr>
            <td>{ employee.firstName }</td>
            <td>{ employee.lastName }</td>
            <td>{ employee.description }</td>
        </tr>
    );
}

export default Employee;

File: spring-boot-react-master/spring-boot-react-master/spring-boot-react/src/main/java/com/springexample/reactdemo/entity/Employee.java
package com.springexample.reactdemo.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Employee {
	
	private @Id @GeneratedValue Long id;
	private @Column String firstName;
	private @Column String lastName;
	private @Column String description;

}
